src/main/resources/atlassian-plugin.xml

```diff
@@ -89,6 +89,11 @@
         <resource type="download" name="highlight.min.css" location="css/highlight.min.css" />
     </web-resource>
     
+    <web-resource key="mermaidjs" name="mermaid.js" >
+        <resource type="download" name="mermaid.min.js" location="js/mermaid.min.js" />
```

Comment:
[nitpick]: these files should have the version in file name, so we know which version we're using, eg: mermaid-1.0.1.min.js